backend/test/akvo/lumen/lib/transformation/derive/js_engine_test.clj

```diff
@@ -1,16 +1,72 @@
 (ns akvo.lumen.lib.transformation.derive.js-engine-test
   (:require
+   [akvo.lumen.lib.transformation.derive :as derive]
    [akvo.lumen.lib.transformation.derive.js-engine :as js-engine]
    [clojure.edn :as edn]
    [clojure.string :as string]
-   [clojure.test :refer [deftest is]]))
+   [clojure.test :refer [deftest is]]
+   [clojure.walk :as walk]))
 
 
 (deftest javascript-logic
   (is (= (js-engine/eval* "1 + 1") 2))
   (is (= (js-engine/eval* "'akvo'.toUpperCase();") "AKVO"))
   (is (not (js-engine/eval* "NaN === NaN;"))))
 
+(deftest row-transform-fn-rqg
+  (let [columns [{"groupId" nil,
+                  "key" true,
+                  "groupName" nil,
+                  "sort" nil,
+                  "direction" nil,
+                  "metadata" nil,
+                  "title" "Identifier",
+                  "type" "text",
+                  "hidden" false,
+                  "columnName" "identifier"}
+                 {"groupId" "567919211",
+                  "groupName" "Repeating Biodata",
+                  "sort" nil,
+                  "direction" nil,
+                  "metadata"
+                  {"columns"
+                   [{"id" "c587209169",
+                     "type" "text",
+                     "title" "Name",
+                     "groupId" nil,
+                     "metadata" nil,
+                     "groupName" nil}
+                    {"id" "c565649136",
+                     "type" "number",
+                     "title" "Age repeated",
+                     "groupId" nil,
+                     "metadata" nil,
+                     "groupName" nil}
+                    {"id" "c569709126",
+                     "type" "number",
+                     "title" "ID number",
+                     "groupId" nil,
+                     "metadata" nil,
+                     "groupName" nil}]},
+                  "title" "Repeating Biodata",
+                  "type" "rqg",
+                  "hidden" false,
+                  "columnName" "c567919211"}]
+        adapter (comp (js-engine/rqg columns)
+                      (js-engine/column-name->column-title columns))
```

Comment:
While this is how we're using it, is not the _unit_ to test? we only want/need to test the `js/rqg` _adapter_.